src/mlpack/methods/ann/ffn_impl.hpp

```diff
@@ -547,25 +545,15 @@ template<typename OutputLayerType, typename InitializationRuleType,
          typename... CustomLayers>
 template<typename Archive>
 void FFN<OutputLayerType, InitializationRuleType, CustomLayers...>::serialize(
-    Archive& ar, const unsigned int version)
+    Archive& ar, const unsigned int /* version */)
```

Comment:
I believe we can remove all the `const unsigned int` parameters now too. :+1:

Reply:
Are you sure that the change in the signature will pass the sfinae?

Reply:
Ah, right, the SFINAE check probably needs to be adapted too.  But cereal doesn't require the `const unsigned int`, so there's no reason to keep it. :+1:

Reply:
Perfect. :+1: :+1: